### PR TITLE
Feature/4 plant catalogue screen

### PR DESCRIPTION
**Pull request by katarzynasitarz**

### CODE REVIEW

File: src/components/AddForm.js [hunk 1]
```diff
@@ -42,81 +37,35 @@ export class AddForm extends Component {
...
-              imageName: ""
-            });
-            alert("Dodano roślinkę!");
-          });
+
+    const plantsRef = firebase.database().ref("plants");
+    const plant = {
+      name: this.state.text,
+      description: this.state.textarea,
+      date: this.state.date,
```
Comment (cytrowski) [inline src/components/AddForm.js]: I recommend using Firebase server time: https://stackoverflow.com/questions/24202641/how-can-you-get-the-server-time-from-firebase

Comment (cytrowski) [reply src/components/AddForm.js]: Here is the official doc: https://firebase.google.com/docs/reference/js/firebase.database.ServerValue

Comment (katarzynasitarz) [reply src/components/AddForm.js]: Ok, I changed it.

Comment (cytrowski) [reply src/components/AddForm.js]: You did or you will? :)

Comment (cytrowski) [reply src/components/AddForm.js]: I approve the PR upfront - nothing more from me here :)

**Pull request merged**